projects/core/src/occ/adapters/cart/occ-cart-voucher.adapter.spec.ts

```diff
@@ -0,0 +1,112 @@
+import {

```

Comment:
This file need to be refactored.
Refer to https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/4555b2c9b0bea7dc95a28dcf8490b06d7c074513/projects/core/src/occ/adapters/user/occ-user-notification-preference.adapter.spec.ts

Reply:
Done